apps/frontend/core-dapp/e2e/utils/limits.ts

```diff
@@ -0,0 +1,41 @@
+import { ethers } from 'ethers'
+import { NETWORKS } from 'prepo-constants'
+import { getContractAddress } from 'prepo-utils'
+import { parseEther } from 'ethers/lib/utils'
+import { bnToBytesArray32, overrideState } from './tenderly'
+import { DepositRecordSlots } from './contract-slots'
+import { supportedContracts } from '../../src/lib/supported-contracts'
+import { DepositRecordAbi__factory } from '../../generated/typechain'
+
+export async function increaseDepositLimits({ rpcUrl }: { rpcUrl: string }): Promise<void> {
+  const provider = new ethers.providers.JsonRpcProvider(rpcUrl, NETWORKS.arbitrumOne)
+
+  const depositRecordAddress = getContractAddress(
+    'DEPOSIT_RECORD',
+    'arbitrumOne',
+    supportedContracts
+  )
+
+  if (!depositRecordAddress) throw new Error("Can't find DepositRecord")
+
+  const depositRecord = DepositRecordAbi__factory.connect(depositRecordAddress, provider)
+
+  const [[globalCapBefore], [userCapBefore]] = await Promise.all([
+    depositRecord.functions.getGlobalNetDepositCap(),
+    depositRecord.functions.getUserDepositCap(),
+  ])
+
+  await overrideState({
+    contract: depositRecordAddress ?? '',
+    provider,
+    slot: DepositRecordSlots._globalNetDepositCap,
+    value: bnToBytesArray32(globalCapBefore.add(parseEther('100'))),
+  })
+
+  await overrideState({
+    contract: depositRecordAddress ?? '',
+    provider,
+    slot: DepositRecordSlots._userNetDepositCap,
+    value: bnToBytesArray32(userCapBefore.add(parseEther('100'))),
+  })
```

Comment:
and here you'd just do:
```suggestion
  await overrideState({
    contract: depositRecordAddress ?? '',
    provider,
    slot: DepositRecordSlots._globalNetDepositCap,
    value: globalCapBefore.add(parseEther('100')),
  })

  await overrideState({
    contract: depositRecordAddress ?? '',
    provider,
    slot: DepositRecordSlots._userNetDepositCap,
    value: userCapBefore.add(parseEther('100')),
  })
```